controllers/articles.controllers.js

```diff
@@ -76,3 +76,28 @@ exports.getArticles = (request, response, next) => {
         })
         .catch(next);
 };
+
+exports.getCommentsByArticleId = (request, response, next) => {
+    const { article_id } = request.params;
+
+    if (isNaN(article_id) || !Number.isInteger(Number(article_id))) {
+        return response.status(400).send({ message: "Invalid ID" });
+    }
+
+    const id = Number(article_id);
```

Comment:
Do you need to coerce this into a number, if it is going to be used within an SQL query string?